queue_services/entity-emailer/src/entity_emailer/email_processors/cease_receiver_notification.py

```diff
@@ -0,0 +1,148 @@
+# Copyright © 2025 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Email processing rules and actions for Cease Receiver notifications."""
+import base64
+import re
+from http import HTTPStatus
+from pathlib import Path
+
+import requests
+from entity_queue_common.service_utils import logger
+from flask import current_app
+from jinja2 import Template
+from legal_api.models import Business, Filing
+
+from entity_emailer.email_processors import (
+    get_filing_document,
+    get_filing_info,
+    get_recipient_from_auth,
+    substitute_template_parts,
+)
+
```

Comment:
This creates basic email with outputs. I will update this once we have finalized template.